.env.production

```diff
@@ -1,4 +1,5 @@
 REACT_APP_SENTRY_DSN=https://60b48d27af18468bbf7633b36c2fe0aa@o676708.ingest.sentry.io/5832897
+REACT_APP_MINIMA_KEY=89a53bcb-1998-475a-9b4d-49aed4daa79e
```

Comment:
this is still checked in. Please generate a new key and don't check it into this PR.